src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import ArticleCard from "./ArticleCard";
+import axios from "axios";
+
+const ArticlesList = () => {
+  const [currentArticles, setCurrentArticles] = React.useState([]);
+
+  React.useEffect(() => {
+    axios
+      .get("https://northcoders-news-7ei3.onrender.com/api/articles")
+      .then(({ data }) => {
+        setCurrentArticles(data.articles);
+      });
+  }, []);
+
+  return (
```

Comment:
Please include a loading pattern to improve the user experience while they’re waiting for the page to render.